### DISCUSSION TITLE
How to show colorscheme name in statusline?

### DISCUSSION THREAD

**Discussion by otakutyrant** [neovim/neovim#24247] [Q&A] [answered]
I tried `vim.opt.statusline = %{colorscheme}`, but it didn't work.

**Comment by zeertzjq** [comment, marked as answer]
`vim.opt.statusline = "%{get(g:, 'colors_name', 'default')}"`

**Reply by otakutyrant** [reply]
Thank you very much!

**Reply by otakutyrant** [reply]
It is confusing that accepting answers and closing discussions are not equivalent. When is it necessary to close the discussion?
